src/tests/ftest/util/server_utils.py

```diff
@@ -34,8 +34,10 @@
 import signal
 import fcntl
 import errno
+import getpass
 import yaml
 
+from ClusterShell.Task import task_self
```

Comment:
(pylint-import-error) Unable to import 'ClusterShell.Task'

Reply:
Hrm.  I wonder why we are still getting this.
https://github.com/daos-stack/daos/blob/4a66506d77d11a701389096757387e1fed0c6882/utils/docker/Dockerfile.centos.7#L66-L69
should be ensuring that `ClusterShell.Task` is available for _pylint_ to import.